src/home/homework-luks.c

```diff
@@ -1633,10 +1632,9 @@ int home_deactivate_luks(UserRecord *h, HomeSetup *setup) {
                 r = sym_crypt_deactivate_by_name(setup->crypt_device, setup->dm_name, 0);
                 if (ERRNO_IS_DEVICE_ABSENT(r) || r == -EINVAL) {
                         log_debug_errno(r, "LUKS device %s is already detached.", setup->dm_node);
-                        we_detached = false;
-                } else if (r < 0)
+                } else if (r < 0) {
                         return log_info_errno(r, "LUKS device %s couldn't be deactivated: %m", setup->dm_node);
-                else {
+                } else {
```

Comment:
Also here. Please drop unnecessary `{}`.

Reply:
I though the first `if` is using `{}` even though it has one call, so though of keeping it consistent.